assets/package.json

```diff
@@ -18,18 +17,19 @@
   },
   "devDependencies": {
     "@babel/core": "^7.3.4",
-    "@babel/polyfill": "^7.2.5",
     "@babel/preset-env": "^7.3.4",
     "babel-loader": "^8.0.5",
     "copy-webpack-plugin": "^4.5.0",
     "css-loader": "^0.28.10",
     "expose-loader": "^0.7.5",
     "extract-text-webpack-plugin": "^4.0.0-beta.0",
-    "jasmine": "^3.3.1",
     "jest": "^24.5.0",
     "style-loader": "^0.20.2",
     "wait-for-expect": "^1.1.0",
     "webpack": "4.0.0",
     "webpack-cli": "^2.0.10"
+  },
+  "jest": {
+    "testRegex": "/__tests__/.*_test\\.js$"
```

Comment:
Added this so only files in the `__tests__` directory ending in `_test.js` are run by Jest. Even though we backed it out, this allows for us to have `helper.js` and similar files.